src/components/TransactionSender/index.ts

```diff
@@ -18,12 +19,17 @@ import {
 } from 'types/enums';
 import { setNonce } from 'utils';
 
-const TransactionSender = () => {
+export interface TransactionSenderType {
+  sendSignedTransactionsAsync?: SendSignedTransactionsAsyncType;
+}
+
+export const TransactionSender = ({
+  sendSignedTransactionsAsync: customSendTxAsync
```

Comment:
why the name change? can't we use the default name, sendSignedTransactionsAsync?

Reply:
sendSignedTransactionsAsync is used below, name change is to differentiate between default and custom

Reply:
TBD